pkg/parser/parser.go

```diff
@@ -12,11 +12,21 @@ import (
 	"github.com/yuin/goldmark/parser"
 	"github.com/yuin/goldmark/renderer/html"
 	"go.abhg.dev/goldmark/wikilink"
+
```

Comment:
Here, on line 23, and in  the command.Flags - no need to separate your code from the rest with an empty line